src/js/api/config.js

```diff
@@ -0,0 +1,23 @@
+if (typeof require !== "undefined") {
+    // eslint-disable-next-line no-redeclare
+    var base = require("../base");
+    // eslint-disable-next-line no-redeclare
+    var ripe = base.ripe;
+}
+
+ripe.Ripe.prototype.resolveProductId = function(productId, options, callback) {

```

Comment:
name this **configResolve**, we need to start making the API methods respect a standard in the naming to avoid collision of names